src/components/Maps/Marker.tsx

```diff
@@ -22,25 +34,36 @@ const Marker = ({
   lng,
   /* eslint-enable */
   isCurrentLocation,
-  otherClicked,
+  type,
+  clicked,
   onClick,
 }: {
   lat: number;
   lng: number;
   isCurrentLocation?: boolean;
-  otherClicked?: boolean;
+  type?: PinType;
```

Comment:
shouldn't this be required not optional?

Reply:
We don't need this field when we are visualizing the current location pin. The current location pin does not have a type because it is not stored in the DB.